misc/scripts/update_GHpages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Following script copies the markdown files from vmware:master to gh-pages
+
+# get latest changes from master
+git pull
+
+#list files with .md file type
+list_files=$( git ls-files "*.md" )
+
+#Create Temp directory to store .md files
+mkdir ~/backup
+echo $list_files
+while IFS=' ' read -ra arr; do
+      for i in "${arr[@]}"; do
+          # process "$i"
+          echo $i
+          echo "Copying file"
+          cp $i ~/backup/
+      done
+ done <<< "$list_files"
+
+#Creating branch gh-pages
+git checkout -b gh-pages origin/gh-pages
+
+#copying md files from backup to jekyll-docs directory
+cp -r ~/backup/* jekyll-docs/
```

Comment:
can you please share the outcome of this one? I am suspecting there will be more files than required and copying to `gh-pages` branch.